releasenotes/notes/feature-sci-git-metadata-0b418d0a3529d11f.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    sci: Add git metadata from tags and from package support
```

Comment:
Can we reword to this release note to include the new tags we plan to send and the environment variables they should set to use this feature. If this gets too wordy we can include a link to ddtrace library docs or datadog's documentation pages.